grammars/css.cson

```diff
@@ -20,294 +20,1218 @@
     'include': '#comment-block'
   }
   {
+    'include': '#escapes'
+  }
+  {
+    'include': '#combinators'
+  }
+  {
     'include': '#selector'
   }
   {
-    'begin': '(@)charset\\b'
-    'captures':
+    'include': '#at-rules'
+  }
+  {
+    'include': '#rule-list'
+  }
+  {
+    'match': ';'
+    'name': 'punctuation.terminator.rule.css'
+  }
+  {
+    'match': '}'
+    'name': 'punctuation.section.end.css'
+  }
+]
+'repository':
+  'at-rules':
+    'patterns': [
+      {
+        # @charset
+        'begin': '\\A(?:\\xEF\\xBB\\xBF)?(?i:(?=\\s*@charset\\b))'
```

Comment:
What are the `\x` escape sequences doing?  Trying to find some reference to those in the specs but I can't.